From 88f761c54596fa22bb670ddddabe491d9135c7b0 Mon Sep 17 00:00:00 2001
From: Cedric Bondier <cedricx.bondier@intel.com>
Date: Mon, 7 Jan 2013 16:15:11 +0100
Subject: [PORT FROM MAIN][BT] Multidevice support: change bluedroid conf and output folder

BZ: 76413

Changing bluedroid configuration for several reasons:
 - Change compilation flags
 - Change output folder to /data/bluedroid
 - Remove deprecated compilation flags
 - Disable Low Power Mode for Broadcom chip

Change-Id: I6e91c1a46bb0c5cc2b010018cd6aaf39da218088
Orig-Change-Id: I342deef0a0157e17ad8c37fea45b5e2017af0a55
Feature: BT
Category: Integration
Purpose: ABSP-Adapter
Origin: Internal
Upstream-candidate: no
Signed-off-by: Raffaele Aquilone <raquiloX>
Reviewed-on: http://android.intel.com:8080/85373
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 Android.mk           |    2 +-
 bta/Android.mk       |    4 ----
 btif/src/btif_core.c |    2 +-
 conf/bt_stack.conf   |    2 +-
 gki/Android.mk       |    5 -----
 stack/Android.mk     |    5 -----
 6 files changed, 3 insertions(+), 17 deletions(-)

diff --git a/Android.mk b/Android.mk
index b5fe1d8..5f2d274 100644
--- a/Android.mk
+++ b/Android.mk
@@ -13,4 +13,4 @@ include $(call all-subdir-makefiles)
 
 # Cleanup our locals
 bdroid_C_INCLUDES :=
-bdroid_CFLaGS :=
+bdroid_CFLAGS :=
diff --git a/bta/Android.mk b/bta/Android.mk
index 7d1c19e..3713ca9 100644
--- a/bta/Android.mk
+++ b/bta/Android.mk
@@ -4,10 +4,6 @@ LOCAL_PATH:= $(call my-dir)
 
 include $(CLEAR_VARS)
 
-ifeq ($(BOARD_HAVE_BLUETOOTH_BCM),true)
-LOCAL_CFLAGS += \
-	-DBOARD_HAVE_BLUETOOTH_BCM
-endif
 LOCAL_CFLAGS += -DBUILDCFG $(bdroid_CFLAGS)
 
 LOCAL_PRELINK_MODULE:=false
diff --git a/btif/src/btif_core.c b/btif/src/btif_core.c
index ca39cc8..8602d3d 100644
--- a/btif/src/btif_core.c
+++ b/btif/src/btif_core.c
@@ -580,7 +580,7 @@ void btif_enable_bluetooth_evt(tBTA_STATUS status, BD_ADDR local_bd)
 
     bte_main_postload_cfg();
 #if (defined(HCILP_INCLUDED) && HCILP_INCLUDED == TRUE)
-    bte_main_enable_lpm(TRUE);
+    bte_main_enable_lpm(FALSE);
 #endif
     /* add passing up bd address as well ? */
 
diff --git a/conf/bt_stack.conf b/conf/bt_stack.conf
index 2ca20ea..1a5a7eb 100644
--- a/conf/bt_stack.conf
+++ b/conf/bt_stack.conf
@@ -3,7 +3,7 @@
 BtSnoopLogOutput=false
 
 # BtSnoop log output file
-BtSnoopFileName=/sdcard/btsnoop_hci.log
+BtSnoopFileName=/data/misc/bluedroid/btsnoop_hci.log
 
 # Enable trace level reconfiguration function
 # Must be present before any TRC_ trace level settings
diff --git a/gki/Android.mk b/gki/Android.mk
index 0ece25b..7e60255 100644
--- a/gki/Android.mk
+++ b/gki/Android.mk
@@ -13,11 +13,6 @@ LOCAL_C_INCLUDES:= $(LOCAL_PATH)/common \
 
 LOCAL_CFLAGS += -Werror $(bdroid_CFLAGS)
 
-ifeq ($(BOARD_HAVE_BLUETOOTH_BCM),true)
-LOCAL_CFLAGS += \
-	-DBOARD_HAVE_BLUETOOTH_BCM
-endif
-
 LOCAL_PRELINK_MODULE:=false
 LOCAL_SRC_FILES:= \
     ./ulinux/gki_ulinux.c \
diff --git a/stack/Android.mk b/stack/Android.mk
index f677ad2..78ac0a5 100755
--- a/stack/Android.mk
+++ b/stack/Android.mk
@@ -33,11 +33,6 @@ LOCAL_C_INCLUDES:= . \
 
 LOCAL_CFLAGS += $(bdroid_CFLAGS)
 
-ifeq ($(BOARD_HAVE_BLUETOOTH_BCM),true)
-LOCAL_CFLAGS += \
-	-DBOARD_HAVE_BLUETOOTH_BCM
-endif
-
 LOCAL_PRELINK_MODULE:=false
 LOCAL_SRC_FILES:= \
     ./a2dp/a2d_api.c \
-- 
1.7.4.1

